plugins/jobs/plugin.zsh

```diff
@@ -0,0 +1,14 @@
+geometry_prompt_jobs_setup() {
+  return true
```

Comment:
I agree. There are a number of rules we are starting to define that maybe would justify a small `CONTRIBUTING.md` file. What do you think?